collector/container/konflux.Dockerfile

```diff
@@ -97,23 +97,11 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS unpacker
 RUN microdnf install -y unzip findutils
```

Comment:
unzip was removed in https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/merge_requests/2624/diffs#dd9d919ac5c33b0feea41d864b4894e4cb6dd1f6_88_88, is it still required here?